internal/monitors/collectd/elasticsearch/elasticsearch.go

```diff
@@ -36,6 +36,8 @@ type Config struct {
 	Port                 uint16 `yaml:"port" validate:"required"`
 	// AdditionalMetrics to report on
 	AdditionalMetrics []string `yaml:"additionalMetrics"`
+	// Cluster name to which the node belongs
```

Comment:
I would make this a bit more descriptive and clarify that it is optional and can be any value they want and that it is used to populate the plugin_instance dimension.

Reply:
Updated the description to be more verbose.